tests/operator_test.go

```diff
@@ -659,6 +659,13 @@ spec:
 		ensureShasums()
 	})
 
+	Describe("[testid:1746]should have created and ready condition", func() {
+		kv := getCurrentKv()
```

Comment:
This has to be in an `It`

Reply:
ah, good catch
and again, there is no `ready` condition anymore, it was renamed to `available`... please change it at all places where you mention ready